envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/SnapshotProperties.kt

```diff
@@ -74,6 +74,9 @@ class SelectorMatching {
 }
 
 class TlsAuthenticationProperties {
+    var plaintextTransportSocketMatchName = "defaultToPlaintext"
```

Comment:
Why it's configurable? I don't see a reason. It only affects metric name, we don't configure any other metric segments, like "ingress_http"

Reply:
I would also change the names to be more consistent with our other stats segments, like "ingress_http", "ingress_secured_routes", so:

enableMTLS -> mtls_match
defaultToPlaintext -> plaintext_match

(I also changed naming to more consistent IMO, but the main point is to use lower case and underscore)

This way metrics:
```
cluster.echo2.defaultToPlaintext.total_match_count
cluster.echo2.enableMTLS.total_match_count
```
will become:
```
cluster.echo2.plaintext_match.total_match_count
cluster.echo2.mtls_match.total_match_count
```